docs/pages/tutorials/minimal/deploy.mdx

```diff
@@ -0,0 +1,280 @@
+
+To use a third party blockchain you need:
+
+- URLs for both HTTP and WebSocket.
+- Address (and the private key for it)
+- ETH on that address.
+  In the case of a test blockchain, there is usually a faucet you can use.
+
+### Deploy the contracts
+
+1. Edit the environment file for the contracts.
+
+   ```sh filename="packages/contracts/.env" copy
+   PRIVATE_KEY=<your private key>
+   ```
+
+1. Add a section to the Foundry configuration file:
+
+   ```toml filename="packages/contracts/foundry.toml" copy
+   [profile.mychain]
+   eth_rpc_url = " <your HTTPS URL goes here> "
+   ```
+
+1. Deploy the contracts.
+
+   ```sh copy
+   cd packages/contracts
+
+   pnpm mud deploy --profile=mychain
+   cd ../..
+   ```
+
+### Run the user interface
+
+1. Edit the supported chains list to add the blockchain you use.
+   To know get the blockchain name to import, [see the list of supported blockchains](https://github.com/wagmi-dev/viem/blob/main/src/chains/index.ts).
+   If your blockchain is not supported, [see the directions on creating a custom blockchain](https://viem.sh/docs/clients/chains.html#custom-chains).
+
+   ```ts filename="packages/client/src/mud/supportedChains.ts" copy showLineNumbers {2,4-5,8}
+   import { MUDChain, latticeTestnet, mudFoundry } from "@latticexyz/common/chains";
+   import { <network> } from "viem/chains";
+
+   <network>.rpcUrls.default.http = [ '<URL to access the blockchain through HTTP(S)>' ]
+   <network>.rpcUrls.default.webSocket = [ '<URL to access the blockchain through WebSocket>' ]
+
+   // If you are deploying to chains other than anvil or Lattice testnet, add them here
+   export const supportedChains: MUDChain[] = [mudFoundry, latticeTestnet, sepolia];
+   ```
+
+1. Specify the chain ID for the blockchain you are using
+
+   ```sh copy
+   export VITE_CHAIN_ID=<chain ID for the blockchain you use>
+   ```
+
+   Note that you can also do this from the browser, by appending `?chainID=<chainID>` to the user interface URL.
+
+1. Run the user interface.
+
+   ```sh copy
+   cd packages/clients
+   pnpm vite
+   ```
+
+1. Browse to the URL for the application.
+
+### Use the user interface
+
+You can already read information (the value of `Counter`, for example) with the user interface.
+However, to issue transactions, for example to increment the counter, you need ETH.
+There are two options here:
+
+#### Application-managed address
+
+This is the default setting for the template.
+The application creates a private key, and uses that to interact with the blockchain.
+The advantage is that as there is no wallet software, the user does not need to manually approve each transaction.
+However, this means that the transactions are issued by a random account with no ETH.
+
+To enable the use of a blockchain that does have gas costs, get the address of the account from the MUD dev tools and send it some testnet ETH.
+
+![](../../get-addr-mud-dev-tools.png)
+
+#### Wallet-managed address
```

Comment:
Note for context: we'll hopefully have a better default client setup with support for external wallets and for creating session wallets soon.